modules/misc/qt.nix

```
@@ -44,19 +44,68 @@ in {
           </variablelist>
         '';
       };
+
+      style = mkOption {
+        type = types.nullOr (types.enum [
+          "adwaita"
+          "adwaita-dark"
+          "cleanlooks"
+          "gtk2"
+          "motif"
+          "plastique"
+        ]);
```

What if the user installed their own qt theme?

```suggestion
        type = types.nullOr types.str;
```

The documentation for available options is already in the description.